### PR TITLE
follow parent project

### LINKED ISSUE #12: Future is now based on Executor instead of ExecutorService
My latest changes on Vavr's Future broke the build.
I will try to fix that...

screen shot 2017-12-28 at 17 28 55

### PR DESCRIPTION
**Pull request by ruslansennov**
fixes #12

### CODE REVIEW

File: src/main/resources/super/io/vavr/concurrent/FutureImpl.java [hunk 1]
```diff
@@ -148,22 +137,17 @@ private FutureImpl(ExecutorService executorService, Option<Try<T>> value, Queue<
...
-                            }
-                        });
-            }
+            this.cancelled = tryComplete(Try.failure(new CancellationException()));
         }
         return this;
     }
 
+    private void updateThread() {
+    }
```
Comment (danieldietrich) [inline src/main/resources/super/io/vavr/concurrent/FutureImpl.java:146]: a private side-effect that does nothing :)

**Review by danieldietrich — APPROVED** [review]
Looks great, thank you!

File: src/main/resources/super/io/vavr/concurrent/FutureImpl.java (hunk 1 above)
Comment (ruslansennov) [reply src/main/resources/super/io/vavr/concurrent/FutureImpl.java:146]: looks like me :smile:

Comment (danieldietrich) [reply src/main/resources/super/io/vavr/concurrent/FutureImpl.java:146]: yes 😁

**Pull request merged**